packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.spec.ts

```diff
@@ -78,7 +78,7 @@ describe('Design Token Renderer', () => {
       expect(writeFile).toHaveBeenCalledTimes(2);
     });
 
-    describe('the comparator', () => {
+    describe('the transformers sorting by name', () => {
```

Comment:
```suggestion
    describe('the transformers sorted by name', () => {
```
not sure about this suggestion?

Reply:
I would leave it as is, you read this methods as "It describes the transformers sorting by name", I am not sure "It describes the transformers sorted by name" is exactly what we mean here.